Exercise_B/index.html

```diff
@@ -1,34 +1,64 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Flower Shop BS</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
 
-    <!-- Link and CDN required for bootstrap -->
+<head>
+  <meta charset="utf-8" />
+  <title>Flower Shop BS</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
 
-    <!-- Google Font - Lato -->
-    <link
-      href="https://fonts.googleapis.com/css?family=Lato"
-      rel="stylesheet"
-    />
-    <!-- Custom Stylesheet -->
-    <link rel="stylesheet" href="./assets/style/stylesheet.css" />
-  </head>
+  <!-- Link and CDN required for bootstrap -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+  <!-- Google Font - Lato -->
+  <link href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
+  <!-- Custom Stylesheet -->
+  <link rel="stylesheet" href="./assets/style/stylesheet.css" />
+</head>
 
-  <body>
-    <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
+<body>
+  <!--Header (flower shop logo block element)-->
+  <header>
+    <img src='assets/images/logo.png'>
+  </header>
+  <!--Banner Jumbotron of the image-->
+  <div class="container-fluid">
+    <div class="row">
+      <div id="bg"></div>
+      <div id="mask"></div>
+      <h1 id="slogan">Beautiful flowers</h1>
+      <a class="btn btn-warning" id="learnMoreBtn">Learn more</a>
+    </div>
+  </div>
 
-    <!--Four containers with img / text-->
-    <!-- 4 containers -->
+  <div class="container-fluid">
+    <div class="row" id="contact">
+      <div class="col-sm-12 col-md-6" id="flower">
+        <img src="/FTWD_Web_Day_03/Exercise_B/assets/images/flower-icon.png" alt="flower">

```

Comment:
For the Image, as you are already in the Exercise B folder you do not have the main folder, as the images are inside the assets folder. 
To specify the image you can change it to this <img src="assets/images/flower-icon.png" alt="flower" /> for the flower icon image.